src/python/WMCore/WMSpec/Steps/Executors/LogCollect.py

```diff
@@ -63,18 +64,24 @@ def execute(self, emulator=None):
         overrideCatalog = getattr(self.step.application, 'overrideCatalog', None)
 
         overrides = {}
+        params = {}
         if hasattr(self.step, 'override'):
             overrides = self.step.override.dictionary_()
+            if 'siteLocalConfig' in overrides.keys():
```

Comment:
@germanfgv I understand and agree with you this is the best way of setting the parameter through which we can overwrite the logcollect destination. But since this config section is not used solely by T0. (unified is also using it), what I would suggest (and what I was actually trying to express during the meeting) is to set an additional Bool flag  notifying the trigger of the new component behavior like: 

```
if 'redirectLogcollect' in overrides.keys() and overrides['redirectLogcollect']: 
```
(suggesting the flag to trigger the new component behavior to be called `redirectLogcollect`)

Reply:
and then with an additional parameter to notify the method to be used for this redirection: 
```
if 'redirectLogcollect' in overrides.keys() and overrides['redirectLogcollect']: 
    if 'useSiteLocalConfig' in overrides.keys() and overrides['useSiteLocalConfig']: 
        < ... new behaviour here.. >
    elif 'useStaticConfig' in overrides.keys():
        < ...some new  behavior here triggered by the 'useStaticConfig' parameter... > 
         ...
```

Reply:
@todor-ivanov With the current master-branch code, it is already possible to redirect the logs. The current method is simply to specify the stageout params in the overrides dictionary, without bolean flags (the presence the parameters is used as the _flag_). The introduction of `redirectLogcollect` would modify the behavior for other users of this code. 

What I am doing here is to use the presence of `siteLocalConfig` in the overrides dict to trigger the new behaviour (The params dict is not necessary and I will remove it). This should in no way modify the regular behaviour, as other users will not set the `siteLocalConfig` key in theri `overrides dictionary`. Am I missing sometnhing?